tests/test_runtime/test_apis.py

```diff
@@ -0,0 +1,54 @@
+import os
+import shutil
+import urllib
+from pathlib import Path
+
+import pytest
+
+from mmdet.apis import init_detector
+
+
+def test_init_detector():
+    project_dir = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
+    project_dir = os.path.join(project_dir, '..')
+
+    config_file = os.path.join(
+        project_dir, 'configs/mask_rcnn/mask_rcnn_r50_fpn_1x_coco.py')
+
+    checkpoint_file = os.path.join(
+        project_dir,
+        'checkpoints/mask_rcnn_r50_fpn_1x_coco_20200205-d4b0c5d6.pth')
+
+    if not os.path.exists(checkpoint_file):
```

Comment:
Downloading the checkpoint during each CI will cost extra fees. We can skip loading checkpoint in the unit tests.

Reply:
I borrowed from here(https://github.com/open-mmlab/mmdetection/blob/bde7b4b7eea9dd6ee91a486c6996b2d68662366d/tests/test_runtime/async_benchmark.py#L39).
But, it looks redundant to me in this unit test. I will remove it